WikiDrips/WikiTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  WikiTableViewController.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/1/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import UIKit
+
+class WikiTableViewController: UITableViewController {
+
+    var wikiDocs = [[WikiDoc]]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+//      TODO: replace with async call to Wikipedia API
+        self.wikiDocs.insert([WikiDoc()], at: 0)
+        self.wikiDocs.insert([WikiDoc()], at: 0)
+        self.wikiDocs.insert([WikiDoc()], at: 0)
```

Comment:
In ObjC the convention is to use `self.` when accessing properties, in Swift it's the opposite, we don't use `self.` unless we have to (For example, in inits and in blocks)